client/routes.js

```diff
@@ -59,7 +54,9 @@ const mapDispatch = (dispatch) => {
   }
 }
 
-export default connect(mapState, mapDispatch)(Routes)
+// The `withRouter` wrapper makes sure that updates are not blocked
+// when the url changes
+export default withRouter(connect(mapState, mapDispatch)(Routes))
```

Comment:
I think `withRouter` was there for the sake of the navbar rather than the routes?

Reply:
It's definitely needed for the routes. Without it, they don't update. The navbar doesn't appear to matter either way since it's not using `NavLink`s or similar.

Reply:
…It was previously in the Main because Main rendered children which included the routes.